### PR TITLE
Feature/post tracking UI

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制]重要報導追蹤-UI/前端: 我的閱讀頁](https://app.asana.com/0/1203699264568808/1207297044447667/f)

# Notice

# Dependency

### CODE REVIEW

File: src/components/my-reading/article-tracking-card.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+`
+
+const LoadingCard = () => {
+  return (
+    <LoadingCardContainer>
+      <div>
+        <Rectangle $width={'48px'} $height={'12px'} />
+      </div>
+      <div>
+        <Rectangle $width={'100%'} $height={'16px'} $marginBottom="wide" />
```
Comment (Aylie-Chou) [inline src/components/my-reading/article-tracking-card.js:109]: Could we add a `enum` to manage $marginBottom type 🙏

File: src/components/my-reading/article-tracking-section.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+const ArticleTrackingContainer = styled.div`
+  width: 100%;
+`
+
+const TrackingCardList = styled.div`
+  /* icon width = 24px, gap = 16px */
+  /*
+  |
+  | <left icon 24px> | <gap 16px> | <cards...> | <gap 16px> | <right icon 24px> |
```
Comment (Aylie-Chou) [inline src/components/my-reading/article-tracking-section.js:34]: good use of comments 👍

File: src/components/my-reading/article-tracking-section.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+                spaceBetween={16}
+                onSwiper={swiper => setSwiper(swiper)}
+                onSlideChange={onSwiperSlideChange}
+                onAfterInit={onSwiperInit}
+              >
+                {trackingCardsJSX(true, cardPerRow)}
+              </Swiper>
+              <ArrowButton
+                $hidden={isSlideEnd}
+                iconComponent={<Arrow direction={Arrow.Direction.RIGHT} />}
```
Comment (Aylie-Chou) [inline src/components/my-reading/article-tracking-section.js:209]: releaseBranch ?

File: src/components/my-reading/article-tracking-section.js [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+            })}
+        </LoadingCardContainer>
+      )}
+      {!isLoading && trackingArticles.length > 0 && (
+        <>
+          <DesktopAndAbove>
+            <TrackingCardList>
+              <ArrowButton
+                $hidden={isSlideBeginning}
+                iconComponent={<Arrow direction={Arrow.Direction.LEFT} />}
```
Comment (Aylie-Chou) [inline src/components/my-reading/article-tracking-section.js:194]: releaseBranch !

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Mostly looking good~~
Please have a look at comments

File: src/components/my-reading/article-tracking-card.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    </LoadingCardContainer>
+  )
+}
+
+export const ArticleTrackingCard = ({
+  isLoading = true,
+  publishDate,
+  trackingTitle,
+  trackingContent,
+  trackingArticleTitle,
```
[the remark this answers is not in the document]
Comment (liruchen32) [reply src/components/my-reading/article-tracking-card.js:133]: fixed

File: src/components/my-reading/article-tracking-section.js (hunk 3 above)
Comment (liruchen32) [reply src/components/my-reading/article-tracking-section.js:209]: fixed

File: src/components/my-reading/article-tracking-section.js (hunk 4 above)
Comment (liruchen32) [reply src/components/my-reading/article-tracking-section.js:194]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**